src/Events/Integrations/Plugins/Tickets_Wallet_Plus/Pdf.php

```diff
@@ -0,0 +1,131 @@
+<?php
+
+namespace TEC\Events\Integrations\Plugins\Tickets_Wallet_Plus;
+
+use Tribe__Template;
+use Tribe__Events__Main as TEC;
+/**
+ * Class Pdf
+ * 
+ * @since TBD
+ * 
+ * @package TEC\Events\Integrations\Plugins\Tickets_Wallet_Plus
+ */
+class Pdf {
+
+		/**
+		 * Template instance.
+		 * 
+		 * @since TBD
+		 * 
+		 * @var \Tribe__Template
+		 */
+		private $template;
+
+		/**
+		 * Get the template.
+		 *
+		 * @since TBD
+		 *
+		 * @return \Tribe__Template
+		 */
+		public function get_template(): Tribe__Template {
+			if ( empty( $this->template ) ) {
+				$template = new Tribe__Template();
+				$template->set_template_origin( TEC::instance() );
+				$template->set_template_folder( 'src/views/integrations/tickets-wallet-plus' );
+				$template->set_template_folder_lookup( true );
+				$template->set_template_context_extract( true );
+				$this->template = $template;
+			}
+			return $this->template;
+		}
+
+		/**
+		 * Filter template vars.
+		 *
+		 * @since TBD
+		 * 
+		 * @param array $vars
+		 * 
+		 * @return array
+		 */
+		public function filter_template_context( $ctx ): array {
```

Comment:
I believe we're always using `$context` as the variable name when referring to the context. In favor of consistency and naming I think it'd be a good idea to use `$context` instead of `$ctx`.